### DISCUSSION TITLE
希望能把夜间模式可以实现的进度条变色功能放到默认模式上

### DISCUSSION THREAD

**Discussion by suheandzl** [the1812/Bilibili-Evolved#1826] [功能建议 / Ideas]
希望能把夜间模式可以实现的进度条变色功能放到默认模式上 因为本人的喜好原因不是太喜欢夜间模式 在偶然的情况下开启了夜间模式 发现进度条变色为主题色的功能非常好 不知道大大能不能考虑一下这个建议

**Comment by Geeyun-JY3** [comment]
有相关 issue：#431

**Reply by suheandzl** [reply]
啊 不好意思才看到 原来人反馈了 不好意思打扰了

**Reply by Geeyun-JY3** [reply]
分拆出来也没啥，毕竟🌚：https://github.com/the1812/Bilibili-Evolved/issues/968#issuecomment-723085022

**Reply by suheandzl** [reply]
啊...哭了
